cypress/e2e/laboratory/preflight.cy.ts

```diff
@@ -1,21 +1,9 @@
-import { dedent } from '../support/testkit';
-
-const selectors = {
-  buttonGraphiQLPreflight: '[aria-label*="Preflight Script"]',
-  buttonModalCy: 'preflight-modal-button',
-  buttonToggleCy: 'toggle-preflight',
-  buttonHeaders: '[data-name="headers"]',
-  headersEditor: {
-    textArea: '.graphiql-editor-tool .graphiql-editor:last-child textarea',
-  },
-  graphiql: {
-    buttonExecute: '.graphiql-execute-button',
-  },
-
-  modal: {
-    buttonSubmitCy: 'preflight-modal-submit',
-  },
-};
+import { dedent } from '../../support/dedent';
+import { cyLaboratory } from './_cy';
+
+const s = cyLaboratory.preflight.selectors;
+
+const cyp = cyLaboratory.preflight;
```

Comment:
If requered to re-assign these namespaces/object whatever `s` and `cyp` are not expressive enough for me. Alternative proposal:
- `selectors`
- `cypress`